### ISSUE TITLE
max keys returned in getkeyrange

### ISSUE THREAD

**Issue by pareshpp** [Seagate/kinetic-protocol#32]
Does the protocol set any limit on max keys returned by server in getkeyrange() operation?

**Comment by jphughes** [comment]
Yes, the limit is 200 keys. The choice of 200 keys was that with 4k keys, the returned protobuf will be <1MByte.

**Comment by pareshpp** [comment]
What if the protobuf size is > 1MB?Let's say if it is 2MB, can the getKeyrange() return 200*2 keys?

**Comment by jphughes** [comment]
The recommendation is to keep the protobuf below 1MB. When you have max length keys (4096), 400 keys is 800kB. We choose 200 because it was round number mostly filling the protobuf.
